docs/markdown/Commands.md

```diff
@@ -149,6 +150,26 @@ optional arguments:
   --clean                               Clean the build directory.
   -C BUILDDIR                           The directory containing build files to
                                         be built.
+  --ninja-args NINJA_ARGS               Arguments to pass to `ninja` (applied
+                                        only on `ninja` backend).
+  --vs-args VS_ARGS                     Arguments to pass to `msbuild` (applied
+                                        only on `vs` backend).
+```
+
+`BACKEND-args` use the following syntax:
+
+If you only pass a single string, then it is considered to have all values separated by commas. Thus invoking the following command:
+
+```
+$ meson compile --ninja-args=foo,bar
+```
+
+would set the value to an array of two elements, `foo` and `bar`.
+
+If you need to have commas in your string values, then you need to pass the value with proper shell quoting like this:
```

Comment:
commas or spaces.

Reply:
Actually not, --ninja-args="a b, c" should be fine and be parsed as `a b` and `c`, right?

Reply:
Fixed

Reply:
> Actually not, --ninja-args="a b, c" should be fine and be parsed as a b and c, right?

Yep, but it still requires additional quoting for the value though?

Reply:
yep, worth mentioning it needs quotes.